cg/server/api.py

```diff
@@ -357,7 +357,7 @@ def options():
             for organism in db.organisms()
```

Comment:
Should we add a pydantic module for organisms? It is currently a dict

Reply:
Good suggestion, but let's do it when we refactor the organism SQL code. I am trying to not extend the scope outside Bed refactoring too much